src/components/templates/layout/index.jsx

```diff
@@ -1,38 +1,36 @@
 import React from 'react';
-import { View, StyleSheet, Platform } from 'react-native';
+import { Platform } from 'react-native';
 import SideMenu from 'react-native-side-menu';
+import styled from '@emotion/native';
+
 import Nav from 'components/organisms/navigation/nav';
 
+const Screen = styled.View`
+  overflow: hidden;
+  width: 100%;
+  height: 100%;
+`;
```

Comment:
Fine for this PR, but for a followup:

A lot of styled elements like this could be usefully turned into re-usable atom components, similar to `Container`.